src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ChangeApprenticeship.feature

```diff
@@ -25,9 +19,33 @@ Scenario: A new apprenticeship has been created as a continuation of an apprenti
 	And the new commitment statement exists in database
 	And the new commitment statement has a new commitments apprenticeship Id
 
-Scenario: Ignore a new apprenticeship has been created as a continuation of an apprenticeship, when we do not know it
-	Given we do not have an existing apprenticeship
+Scenario: Update registration details if the apprentice has not confirmed his identity
+	Given we do have an unconfirmed registration
+	And we have an update apprenticeship request
+	When the update is posted
+	Then there should be no commitment statements in the database
+	And we have updated the apprenticeship details for the unconfirmed registration
+	And registration commitments apprenticeship are updated correctly
+
+Scenario: Update registration details with Continuation Id if the apprentice has not confirmed his identity 
+	Given we do have an unconfirmed registration
 	And we have a update apprenticeship continuation request
 	When the update is posted
-	Then the result should return accepted
-	And there should be no commitment statements in the database
+	Then there should be no commitment statements in the database
+	And we have updated the apprenticeship details for the unconfirmed registration
+	And registration commitments apprenticeship are updated correctly
+
+Scenario: No apprenticeship or registation details found
+	Given we do not have an existing apprenticeship, confirmed or unconfirmed
+	And we have an update apprenticeship request
+	When the update is posted
+	Then the response is bad request
+	And a domain exception is thrown
```

Comment:
Is there a benefit in having a `BadReqeust` response, rather than a `NotFound` one?  I'd have thought that `NotFound` fits better here as that's literally what's happened.

I guess it doesn't make much difference to the overall behaviour - it's an error!

Reply:
Did consider not found, but as you say it's a error.